src/components/SUE/report/SueReportUser.tsx

```diff
@@ -104,7 +106,10 @@ export const SueReportUser = ({
             <Checkbox
               checked={!!value}
               onPress={() => onChange(!value)}
-              link="https://smart-village.app/datenschutzerklaerung/"
+              link={
+                configuration?.limitation?.privacyPolicy?.value ||
+                'https://smart-village.app/datenschutzerklaerung/'
```

Comment:
we have defaults in the code, right. so the hardcoded url should come from that defaults i guess.

Reply:
Are you talking about the default data in `sueDefaultConfigurations`?

Reply:
should be there, or am i wrong? it is the default and fallback for sue here?
do you think it does not fit to the defaults?

Reply:
No, that's right. But I wonder if the default will not be overwritten if there is no data? For example, `privacyPolicy.value = "link"` by default. But if no data comes from the api, won't this default be overwritten? I thought it would be. 😔

Reply:
ok, in this case, we can think about it in a later process when integrating more and more configurations.